docs/config.toml

```diff
@@ -0,0 +1,12 @@
+baseURL = 'http://example.org/'
```

Comment:
@IEvangelist think we need to update this to be the root of the github pages site do you have the full URL?